test/api/get.spec.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-env mocha */
+/* globals apiClients */
+'use strict'
+
+const expect = require('chai').expect
+const isNode = require('detect-node')
+const fs = require('fs')
+const bl = require('bl')
+const concat = require('concat-stream')
+const through = require('through2')
+
+const path = require('path')
+const streamEqual = require('stream-equal')
+
+const extract = require('tar-stream').extract
+
+let testfile
+let testfileBig
+
+if (isNode) {
+  testfile = fs.readFileSync(path.join(__dirname, '/../testfile.txt'))
+  testfileBig = fs.createReadStream(path.join(__dirname, '/../15mb.random'), { bufferSize: 128 })
+} else {
+  testfile = require('raw!../testfile.txt')
+}
+
```

Comment:
Add a comment here to explain that these are the tests for the options supported by go-ipfs, not yet provided in js-ipfs.